interp/constrextern.ml

```diff
@@ -107,18 +107,19 @@ let deactivate_notation nr =
   | SynDefRule kn ->
      (* shouldn't we check whether it is well defined? *)
      inactive_notations_table := IRuleSet.add nr !inactive_notations_table
-  | NotationRule (scopt, ntn) ->
-     match availability_of_notation (scopt, ntn) (scopt, []) with
+  | NotationRule (which, ntn) ->
```

Comment:
The name `which` does not seem very informative.

Reply:
(idem for other occurrences below)

Reply:
Yes, `which` is not a very discriminating word. I thought at `kind` at some time but it would not be better. Maybe `specif` since it is in the type `notation_specificity`, but at the same time, `specificity` is not very "specific" here. The idea is to tell whether it is in some scope or lonely. So, maybe `isinscope`, or `scopekind`? We could also keep the original `scopt` and rename `notation_specificity` into `notation_optionally_in_scope`, or `notation_with_scope`. I'm missing imagination. Maybe would I then choose `scopt` and `notation_with_scope` if noone has better to propose.